processing/src/main/java/org/apache/druid/segment/data/Indexed.java

```diff
@@ -55,8 +55,8 @@
   int indexOf(@Nullable T value);
 
   /**
-   * Returns the index of "value" in some object whose values are accessible by index some {@link IndexedGetter}, or
-   * (-(insertion point) - 1) if the value is not present, in the manner of Arrays.binarySearch.
+   * Returns the index of "value" in some object in the {@link Indexed}, or (-(insertion point) - 1) if the value is
+   * not present, in the manner of Arrays.binarySearch.
```

Comment:
I'd personally prefer that we remove this bit of documentation.  It's an implementation detail that I don't think we should try to enshrine in the docs anymore (especially after this PR)

Reply:
i just killed this method and moved this logic into `GenericIndexed` which was the only caller again after the changes in this PR. I think it is ok for `GenericIndexed.indexOf` to mention that it has this contract for `indexOf`, since it is an implementation detail that the `BitmapIndex` implementation of `StringBitmapIndexColumnPartSupplier` cares about.